getting_started.md

```diff
@@ -287,6 +287,43 @@ use the [analysis runner](https://github.com/populationgenomics/analysis-runner)
 which builds a Batch pipeline from a specific commit in a GitHub repository by
 running the driver itself on Hail Batch.
 
+### Analysis runner
+
+The analysis-runner builds a Batch pipeline from a speific commit in a GitHub repository. You can interact with an analysis server with the CLI. This can be installed with:
+
+```bash
+conda install -c cpg analysis-runner
+```
+
+Make sure you have the name of the dataset, as this corresponds to a specific analysis server and approved repository.
+
+You can use the analysis runner CLI in two ways:
+
+1. Omitting the `--repository` parameter: use the repository of the local directory that you're in: (a) get the repository name from the git remote; (b) use the commit of HEAD (if the `--commit` parameter is omitted); (c) Make the script path relative to the root of the git repository.
```

Comment:
This is pretty detailed and might be a bit overwhelming for new analysts that don't quite understand yet how the whole thing works.

How about just covering the common case here, e.g. "assuming you want to run `python3 myscript.py -p 3`, how do you do this using the analysis runner: commit, then call analysis-runner instead.

For any detailed information (like using another repo) we could just give a link to the documentation in the analysis-server repository. That also avoids duplication and issues with keeping the docs in sync.

Reply:
Awesome, I'll make those changed, with respect to your other comment now :)